prerequisites/software.rst

```diff
@@ -162,6 +162,10 @@ You can choose between the following database servers:
 
    .. include:: /appendix/includes/mysql-deprication-note.rst
 
+.. note::
+   Transaction-based connection pooling is not supported.
+   If your setup is built this way, database migrations may fail.
+
```

Comment:
I think we need to expand this to say that it is for the usage of PgBouncer (see the original issue for details). That seems to be a popular software in cloud environments for Postgresql deployment.
And perhaps we should positively state that instead session connection pooling needs to be used.
See also https://www.pgbouncer.org/features.html

Reply:
I added PgBouncer as an example, I think there are more solutions. I found [this](https://www.rockdata.net/tutorial/configure-connection-pool/#connection-pooling-solutions) where they mention some of them.


Is it ok now? Feel free to request changes again ;-)